ansicolor.js

```diff
@@ -37,91 +37,116 @@ const colorCodes      = [   'black',      'red',      'green',      'yellow',
 
 /*  ------------------------------------------------------------------------ */
 
-const clean = obj => {
-                for (const k in obj) { if (!obj[k]) { delete obj[k] } }
-                return (O.keys (obj).length === 0) ? undefined : obj
-            }
```

Comment:
Remove clean as it changes the "hidden class of the object", it is better to just keep the properties but change their values